docs/openapi3.yaml

```diff
@@ -1383,6 +1387,142 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/error_response'
+  # Webhooks
+  /v1/webhooks:
+    get:
+      operationId: retrieve-webhooks
+      tags:
+        - Webhooks
+      summary: Retrieve webhooks
+      description: Retrieve all webhooks.
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+               schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/webhook'
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error_response'
+        '404':
```

Comment:
the return type is an array
no need for 404
if it's empty the result will be []

Reply:
fixed it